onnxruntime/contrib_ops/cpu/crop_and_resize.cc

```diff
@@ -58,7 +57,7 @@ void CropAndResizeForward(
     T* top_data,
     const std::string& mode,
     const int32_t* batch_indices_ptr,
-    const ThreadPool* ttp) {
+    ThreadPool* ttp) {
   int64_t n_rois = nthreads / channels / pooled_width / pooled_height;
 
```

Comment:
unrelated to your changes, but this seems wrong as it's used as the total. if nthreads == 1 I expect this will be 0, and the code in ThreadPool::ParallelFor would do nothing as this value is passed in as the total number of pieces of work.

Reply:
nthreads is n tasks, not threads.